demo/index.html

```diff
@@ -20,7 +20,14 @@
       <h3>Basic lrnsys-randomimage demo</h3>
       <demo-snippet>
         <template>
-          <lrnsys-randomimage></lrnsys-randomimage>
+          <lrnsys-randomimage images="[
```

Comment:
@kat-wehr you had these hard coded into the element meaning everyone would see them this way. by wiring them into the demo, this lets people get the idea of the element but be able to use it more generally as well.